api/base/views.py

```diff
@@ -533,23 +534,35 @@ def postprocess_query_param(self, key, field_name, operation):
 
     def build_query_from_field(self, field_name, operation):
         if field_name == 'permission':
-            if operation['op'] != 'eq':
-                raise InvalidFilterOperator(value=operation['op'], valid_operators=['eq'])
+            if operation['op'] not in ['eq', 'exact']:
+                raise InvalidFilterOperator(value=operation['op'], valid_operators=['eq', 'exact'])
+
             # operation['value'] should be 'admin', 'write', or 'read'
             query_val = operation['value'].lower().strip()
             if query_val not in API_CONTRIBUTOR_PERMISSIONS:
                 raise InvalidFilterValue(value=operation['value'])
             # This endpoint should only be returning *contributors* not group members
             resource = self.get_resource()
-            if query_val == READ:
-                # If read, return all contributors
-                return Q(user_id__in=resource.contributors.values_list('id', flat=True))
-            elif query_val == WRITE:
-                # If write, return members of write and admin groups, both groups have write perms
-                return Q(user_id__in=(resource.get_group(WRITE).user_set.values_list('id', flat=True) | resource.get_group(ADMIN).user_set.values_list('id', flat=True)))
-            elif query_val == ADMIN:
-                # If admin, return only members of admin group
-                return Q(user_id__in=resource.get_group(ADMIN).user_set.values_list('id', flat=True))
+            if operation['op'] == 'eq':
+                if query_val == READ:
+                    # If read, return all contributors
+                    return Q(user_id__in=resource.contributors.values_list('id', flat=True))
+                elif query_val == WRITE:
+                    # If write, return members of write and admin groups, both groups have write perms
+                    return Q(user_id__in=(resource.get_group(WRITE).user_set.values_list('id', flat=True) | resource.get_group(ADMIN).user_set.values_list('id', flat=True)))
```

Comment:
`resource.get_group(ADMIN).user_set.values_list('id', flat=True)` is making this seem a little convoluted for a single line, try breaking some of these lines into smaller functions with descriptive names. A minor suggestion for otherwise good logic.